doc/isomorph.xml

```diff
@@ -1,14 +1,95 @@
-##                                                         Wilf A. Wilson
+#W  isomorph.xml                                        James D. Mitchell
+#Y  Copyright (C) 2014-17                                  Wilf A. Wilson 
 ##
 ##  Licensing information can be found in the README file of this package.
 ##
 #############################################################################
 ##
 
+<#GAPDoc Label="DigraphsUseNauty">
+<ManSection>
+  <Func Name="DigraphsUseNauty" Arg=""/>
+  <Func Name="DigraphsUseBliss" Arg=""/>
+  <Returns>Nothing.</Returns>
+  <Description>
+    These functions can be used to specify whether &nauty; or &bliss; should be
+    used by default by &Digraphs;. If &NautyTracesInterface; is not available,
+    then these functions do nothing. Otherwise, by calling
+    <C>DigraphsUseNauty</C> subsequent computations will default to using
+    &nauty; rather than &bliss;, where possible.
+    <P/>
+
+    You can call these functions at any point in a &GAP; session, as many
+    times as you like, it is guaranteed that existing digraphs remain
+    valid, and that comparison of existing digraphs and newly created digraphs
+    via 
+    <Ref Oper="IsIsomorphicDigraph" Label="for digraphs"/>,
+    <Ref Oper="IsIsomorphicDigraph" 
+      Label="for digraphs and homogeneous lists"/>, 
+    <Ref Oper="IsomorphismDigraphs" Label="for digraphs"/>, and
+    <Ref Oper="IsomorphismDigraphs" 
+      Label="for digraphs and homogeneous lists"/> are also valid.<P/>
+
+    It is also possible to compute the automorphism group of a specific digraph
+    using both &nauty; and &bliss; using <Ref Attr="NautyAutomorphismGroup"/>
+    and <Ref Attr="BlissAutomorphismGroup"/>, respectively.
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="BlissAutomorphismGroup">
+<ManSection>
+  <Attr Name="BlissAutomorphismGroup" Arg="digraph[, colours]"/>
+  <Returns>A permutation group.</Returns>
+  <Description>
+    If <A>digraph</A> is a digraph, then this attribute contains the group of
+    automorphisms of <A>digraph</A> as calculated using &bliss; by Tommi Junttila
+    and Petteri Kaski. <P/>
+
+    The attribute <Ref Attr="AutomorphismGroup" Label="for a digraph"/> and
+    operation <Ref Attr="AutomorphismGroup" 
+      Label="for a digraph and a homogeneous list"/> returns
+    the value of either <C>BlissAutomorphismGroup</C> or  <Ref
+      Attr="NautyAutomorphismGroup"/>. These groups are, of course, equal but
+    their generating sets may differ. <P/>
+
+    See also <Ref Func="DigraphsUseBliss"/>, and <Ref Func="DigraphsUseNauty"/>.
+
+    <Example><![CDATA[
+gap> BlissAutomorphismGroup(JohnsonDigraph(5, 2));
+Group([ (3,4)(6,7)(8,9), (2,3)(5,6)(9,10), (2,5)(3,6)(4,7), (1,2)(6,8)
+(7,9) ])]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="NautyAutomorphismGroup">
+<ManSection>
+  <Attr Name="NautyAutomorphismGroup" Arg="digraph[, colours]"/>
```

Comment:
Is it true that nauty can only handle digraphs without multiple edges? If so, this should be documented in all the relevant places.